src/main/resources/db/migration/V202409221730__create_feedback_and_feedback_response_tables.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE IF NOT EXISTS feedback (
+                                        id BIGSERIAL PRIMARY KEY,
+                                        company_id BIGINT NOT NULL,
+                                        sender_id BIGINT NOT NULL,
```

Comment:
in case of an anonymous sender, we can't save that id, it violates the anonymity .
So let's remove the `not null` limitation